libraries/AP_Notify/Display.cpp

```diff
@@ -424,7 +424,7 @@ void Display::update_prearm(uint8_t r)
 void Display::update_gps(uint8_t r)
 {
 	static const char * gpsfixname[] = {"Other", "NoGPS","NoFix","2D","3D","DGPS", "RTK f", "RTK F"};
-	char msg [DISPLAY_MESSAGE_SIZE];
+	char msg [DISPLAY_MESSAGE_SIZE - 1];
```

Comment:
It is a specification that the message does not scroll.
In this case, up to 18 characters can be displayed.
Therefore, 18 characters plus NULL character is 19 bytes.